tests/core/test_delegating.py

```diff
@@ -739,6 +739,8 @@ def test_delegables_escrow():
         parsing.Parser().parse(ims=bytearray(gateIcp), kvy=torKvy, local=True)
         assert gateHab.pre not in torKvy.kevers
         assert len(torHab.db.delegables.get(keys=snKey(gateHab.kever.serder.preb, gateHab.kever.serder.sn))) == 1
+        # Exercise the MissingDelegableApprovalError case
+        torKvy.processEscrowDelegables()
```

Comment:
Is there expected to be additional assertions here to catch `MissingDelegableApprovalError`?

Reply:
or should there be additional assertions to verify there _wasn't_ a missing delegable approval error?

Reply:
seems odd to add the processing line but no assertions in the test

Reply:
> Is there expected to be additional assertions here to catch `MissingDelegableApprovalError`?

No, the error is part of the escrow processing (the exception doesn't make it out of the processing).

Reply:
> or should there be additional assertions to verify there _wasn't_ a missing delegable approval error?

The test was valid originally, but it never exercised the MissingDelegableApprovalError so it did discover the logging error. This small addition does a better job of simulating that processEscrowDelegables is being called multiple times during the flow (becuase it is part of asynchronous escrow processing)

Reply:
> seems odd to add the processing line but no assertions in the test

The constantly running escrow processors are difficult to capture in a small test. this is the shortest path to exercising the case when the processor runs before the user managed flow is complete.